config/webhook/manifests.yaml

```diff
@@ -6,17 +6,15 @@ metadata:
   creationTimestamp: null
   name: validating-webhook-configuration
 webhooks:
-- clientConfig:
-    caBundle: Cg==
```

Comment:
`caBundle` seems to be here for historical reason, but it no longer exists in generated by newer version of controller-gen (Please also see https://github.com/kubernetes-sigs/controller-tools/blob/release-0.2/pkg/webhook/parser.go#L223-L226). According to the comment, it would be safe if no one uses with k8s 1.12 or earlier.